### DISCUSSION TITLE
Hello i have some question!

### DISCUSSION THREAD

**Discussion by jhin93** [rainbow-me/rainbowkit#1437] [Q&A] [answered]
Hello Rainbowkit!
Our company uses Rainbowkit for wallet connection to our site.
And there are some issues with Rainbowkit.
It runs with no problem, but there are some warnings popped up in Chrome dev tools console.
it looks like this.
![image](https://github.com/rainbow-me/rainbowkit/assets/55314087/c5f3ae0c-50a4-43ae-a0c9-49c5c5d08f38)

It was estimated the issues about styles, but i'm not sure.
Please tell me if you know or have seen this before.

Have a good day.

**Comment by DanielSinclair** [comment, marked as answer]
@jhin93 I believe that is not related to RainbowKit. It is more likely related to a wallet you have installed as their inpage script is injecting code and can cause warnings like this.
